proto/viam/app/v1/app.proto

```diff
@@ -912,6 +912,15 @@ message RegistryItem {
     ModuleMetadata module_metadata = 11;
     MLModelMetadata ml_model_metadata = 12;
   }
+
+  // The total number of robots using this item outside of the owning org
+  int64 total_external_robot_usage = 13;
+  // The total number of organizations using this item outside of the owning org
+  int64 total_external_organization_usage = 14;
```

Comment:
is `total_external_robot_usage` = `total_robot_usage` - <# of robots in your org using the item>?
similarly `total_external_organization_usage` = `total_organization_usage` - 1?

Reply:
Yup that is correct. There aren't fields for `total_internal_robot/organization_usage` so you can subtract external from total to get internal